<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Introduction to Somatic Variants | Introduction to Clinical Interpretation of Somatic Cancer Variants</title>
  <meta name="description" content="An introduction to the background and basic concepts around cancer variant interpretation" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Introduction to Somatic Variants | Introduction to Clinical Interpretation of Somatic Cancer Variants" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="An introduction to the background and basic concepts around cancer variant interpretation" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Introduction to Somatic Variants | Introduction to Clinical Interpretation of Somatic Cancer Variants" />
  
  <meta name="twitter:description" content="An introduction to the background and basic concepts around cancer variant interpretation" />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/ITN_favicon.ico" type="image/x-icon" />
<link rel="prev" href="introduction.html"/>
<link rel="next" href="introduction-to-somatic-evidence-types-in-cancer.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="assets/style_ITN.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.itcrtraining.org/"><img src="assets/ITN_logo.png" style="padding-left: 15px; padding-top: 8px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About this Course</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#motivation"><i class="fa fa-check"></i><b>1.1</b> Motivation</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#target-audience"><i class="fa fa-check"></i><b>1.2</b> Target Audience</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#curriculum"><i class="fa fa-check"></i><b>1.3</b> Curriculum</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="introduction-to-somatic-variants.html"><a href="introduction-to-somatic-variants.html"><i class="fa fa-check"></i><b>2</b> Introduction to Somatic Variants</a>
<ul>
<li class="chapter" data-level="2.1" data-path="introduction-to-somatic-variants.html"><a href="introduction-to-somatic-variants.html#learning-objectives"><i class="fa fa-check"></i><b>2.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="2.2" data-path="introduction-to-somatic-variants.html"><a href="introduction-to-somatic-variants.html#the-bottleneck-problem-in-cancer-variant-interpretation"><i class="fa fa-check"></i><b>2.2</b> The Bottleneck Problem in Cancer Variant Interpretation</a></li>
<li class="chapter" data-level="2.3" data-path="introduction-to-somatic-variants.html"><a href="introduction-to-somatic-variants.html#sequencing-background"><i class="fa fa-check"></i><b>2.3</b> Sequencing Background</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="introduction-to-somatic-variants.html"><a href="introduction-to-somatic-variants.html#human-genome-reference"><i class="fa fa-check"></i><b>2.3.1</b> Human Genome Reference</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="introduction-to-somatic-variants.html"><a href="introduction-to-somatic-variants.html#common-variant-types"><i class="fa fa-check"></i><b>2.4</b> Common Variant Types</a></li>
<li class="chapter" data-level="2.5" data-path="introduction-to-somatic-variants.html"><a href="introduction-to-somatic-variants.html#variant-representation-and-classification"><i class="fa fa-check"></i><b>2.5</b> Variant Representation and Classification</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="introduction-to-somatic-evidence-types-in-cancer.html"><a href="introduction-to-somatic-evidence-types-in-cancer.html"><i class="fa fa-check"></i><b>3</b> Introduction to Somatic Evidence Types in Cancer</a>
<ul>
<li class="chapter" data-level="3.1" data-path="introduction-to-somatic-evidence-types-in-cancer.html"><a href="introduction-to-somatic-evidence-types-in-cancer.html#learning-objectives-1"><i class="fa fa-check"></i><b>3.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="3.2" data-path="introduction-to-somatic-evidence-types-in-cancer.html"><a href="introduction-to-somatic-evidence-types-in-cancer.html#somatic-clinically-actionable-evidence-types"><i class="fa fa-check"></i><b>3.2</b> Somatic Clinically Actionable Evidence Types</a></li>
<li class="chapter" data-level="3.3" data-path="introduction-to-somatic-evidence-types-in-cancer.html"><a href="introduction-to-somatic-evidence-types-in-cancer.html#somatic-functional-and-oncogenic-evidence-distinction"><i class="fa fa-check"></i><b>3.3</b> Somatic Functional and Oncogenic Evidence Distinction</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="introduction-to-somatic-evidence-types-in-cancer.html"><a href="introduction-to-somatic-evidence-types-in-cancer.html#functional-evidence-and-mullers-morphs"><i class="fa fa-check"></i><b>3.3.1</b> Functional Evidence and Muller’s Morphs</a></li>
<li class="chapter" data-level="3.3.2" data-path="introduction-to-somatic-evidence-types-in-cancer.html"><a href="introduction-to-somatic-evidence-types-in-cancer.html#oncogenic-evidence-and-the-hallmarks-of-cancer"><i class="fa fa-check"></i><b>3.3.2</b> Oncogenic Evidence and The Hallmarks of Cancer</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="somatic-variant-classification-guidelines.html"><a href="somatic-variant-classification-guidelines.html"><i class="fa fa-check"></i><b>4</b> Somatic Variant Classification Guidelines</a>
<ul>
<li class="chapter" data-level="4.1" data-path="somatic-variant-classification-guidelines.html"><a href="somatic-variant-classification-guidelines.html#learning-objectives-2"><i class="fa fa-check"></i><b>4.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="4.2" data-path="somatic-variant-classification-guidelines.html"><a href="somatic-variant-classification-guidelines.html#ampascocap-guidelines"><i class="fa fa-check"></i><b>4.2</b> AMP/ASCO/CAP Guidelines</a></li>
<li class="chapter" data-level="4.3" data-path="somatic-variant-classification-guidelines.html"><a href="somatic-variant-classification-guidelines.html#clingencgcvicc"><i class="fa fa-check"></i><b>4.3</b> ClinGen/CGC/VICC</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="introduction-to-clingen.html"><a href="introduction-to-clingen.html"><i class="fa fa-check"></i><b>5</b> Introduction to ClinGen</a>
<ul>
<li class="chapter" data-level="5.1" data-path="introduction-to-clingen.html"><a href="introduction-to-clingen.html#learning-objectives-3"><i class="fa fa-check"></i><b>5.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="5.2" data-path="introduction-to-clingen.html"><a href="introduction-to-clingen.html#clingen-somatic"><i class="fa fa-check"></i><b>5.2</b> ClinGen Somatic</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="introduction-to-cancer-variant-knowledgebases.html"><a href="introduction-to-cancer-variant-knowledgebases.html"><i class="fa fa-check"></i><b>6</b> Introduction to Cancer Variant Knowledgebases</a>
<ul>
<li class="chapter" data-level="6.1" data-path="introduction-to-cancer-variant-knowledgebases.html"><a href="introduction-to-cancer-variant-knowledgebases.html#learning-objectives-4"><i class="fa fa-check"></i><b>6.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="6.2" data-path="introduction-to-cancer-variant-knowledgebases.html"><a href="introduction-to-cancer-variant-knowledgebases.html#types-of-cancer-variant-knowledgebases"><i class="fa fa-check"></i><b>6.2</b> Types of Cancer Variant Knowledgebases</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="introduction-to-cancer-variant-knowledgebases.html"><a href="introduction-to-cancer-variant-knowledgebases.html#cosmic"><i class="fa fa-check"></i><b>6.2.1</b> COSMIC</a></li>
<li class="chapter" data-level="6.2.2" data-path="introduction-to-cancer-variant-knowledgebases.html"><a href="introduction-to-cancer-variant-knowledgebases.html#clinvar"><i class="fa fa-check"></i><b>6.2.2</b> ClinVar</a></li>
<li class="chapter" data-level="6.2.3" data-path="introduction-to-cancer-variant-knowledgebases.html"><a href="introduction-to-cancer-variant-knowledgebases.html#oncokb"><i class="fa fa-check"></i><b>6.2.3</b> OncoKB</a></li>
<li class="chapter" data-level="6.2.4" data-path="introduction-to-cancer-variant-knowledgebases.html"><a href="introduction-to-cancer-variant-knowledgebases.html#my-cancer-genome"><i class="fa fa-check"></i><b>6.2.4</b> My Cancer Genome</a></li>
<li class="chapter" data-level="6.2.5" data-path="introduction-to-cancer-variant-knowledgebases.html"><a href="introduction-to-cancer-variant-knowledgebases.html#foundation-one"><i class="fa fa-check"></i><b>6.2.5</b> Foundation One</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="introduction-to-the-civic-knowledgebase.html"><a href="introduction-to-the-civic-knowledgebase.html"><i class="fa fa-check"></i><b>7</b> Introduction to the CIViC knowledgebase</a>
<ul>
<li class="chapter" data-level="7.1" data-path="introduction-to-the-civic-knowledgebase.html"><a href="introduction-to-the-civic-knowledgebase.html#learning-objectives-5"><i class="fa fa-check"></i><b>7.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="7.2" data-path="introduction-to-the-civic-knowledgebase.html"><a href="introduction-to-the-civic-knowledgebase.html#civic-evidence"><i class="fa fa-check"></i><b>7.2</b> CIViC Evidence</a></li>
<li class="chapter" data-level="7.3" data-path="introduction-to-the-civic-knowledgebase.html"><a href="introduction-to-the-civic-knowledgebase.html#civic-somatic-assertions"><i class="fa fa-check"></i><b>7.3</b> CIViC Somatic Assertions</a></li>
<li class="chapter" data-level="7.4" data-path="introduction-to-the-civic-knowledgebase.html"><a href="introduction-to-the-civic-knowledgebase.html#introduction-to-the-civic-web-interface"><i class="fa fa-check"></i><b>7.4</b> Introduction to the CIViC web interface</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>8</b> Conclusions</a></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="9" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>9</b> References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by: </a> </p>
<p style="text-align:center;"> <a href="http://jhudatascience.org/"> The Johns Hopkins Data Science Lab </a></p>
<a href="http://jhudatascience.org/"><img src="https://jhudatascience.org/images/dasl.png" style=" width: 80%; filter: grayscale(100%); padding-left: 40px; padding-top: 8px; vertical-align: top "</a>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
<p style="padding-left: 40px;"><div class="trapezoid" style = "padding-left: 40px;"><span>  <a href="https://forms.gle/hc8Xt3Y2Znjb6M4Y7"> Click here to provide feedback</a> <img src="assets/itcr_arrow.png" style=" width: 10%" ></span></div></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Introduction to Clinical Interpretation of Somatic Cancer Variants</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=1.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
  <link href="assets/style_ITN.css" rel="stylesheet">
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/itcr_main_image.png">
</div>
<div id="introduction-to-somatic-variants" class="section level1" number="2">
<h1><span class="header-section-number">Chapter 2</span> Introduction to Somatic Variants</h1>
<div id="learning-objectives" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Learning Objectives</h2>
<p>This chapter will cover:</p>
<ul>
<li>Somatic Variant Interpretation Bottleneck Problem</li>
<li>Next Generation Sequencing Techniques</li>
<li>Human Genome Reference</li>
<li>Common Variant Types</li>
<li>Variant Representation</li>
</ul>
</div>
<div id="the-bottleneck-problem-in-cancer-variant-interpretation" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> The Bottleneck Problem in Cancer Variant Interpretation</h2>
<p>As next generation, massively parallel sequencing of tumor genomes has become integrated into clinical practice, a so-called bottleneck problem has arisen, where a large number of somatic tumor sequence variants need to be interpreted for their potential clinical impact.</p>
<p><img src="resources/images/02-somatic_concepts_files/figure-html/1zwvAVLWpN2mKrJLgo5ZcAuSKQrHZ8zunIYHLe-w6bBo_g22509046821_0_0.png" title="The variant interpretation bottleneck" alt="The variant interpretation bottleneck" width="100%" style="display: block; margin: auto;" /></p>
<p>In order to address this problem, databases containing information on the clinical impact of tumor variants have been created. These databases may be the result of private efforts and siloed for use by a particular group. Some efforts curate knowledgebases available to the community with access fees that go towards funding further curation. Yet another model is to create free public knowledgebases for variant clinical interpretation which leverage public curation and expert moderation.</p>
<p>The biomedical literature has grown exponentially, including literature reporting new variants in cancer. This compounds the variant interpretation bottleneck as the number of variants in need of interpretation is not only large but also ever growing.</p>
<p>The Genomic landscape of many tumor types has been explored, with the number of coding variants in a tumor type ranging from a few to thousands. Individuals carry an inherited diploid genotype, referred to as germline, shared across all cells, containing common and rare variants relative to the human population. Individual cells in the body also develop spontaneous mutations, which are called somatic variants. Regularly dividing cells are especially susceptible to acquiring somatic mutations over the lifetime of an organism. As cells accumulate somatic mutations, they can become susceptible to precancerous or cancerous transformation. This is generally the result of mutations which disrupt the function of Tumor Suppressor genes, which are genes whose protein product functions within cells to regulate and halt the cell cycle, induce cellular differentiation, promote cell death via apoptosis, or other mechanisms which control cellular potential for malignancy. Cancer driving, or oncogenic mutations to tumor suppressors are generally of a reduced function or loss of function (LOF) type.</p>
<p>Mutations may also promote cancer transformation via the activation of oncogenes, which are genes whose protein product is usually involved in promoting cellular growth in a regulated fashion, and can help drive uncontrolled growth when regulation is removed. Oncogenes are often part of growth signaling pathways such as those involving MAP or AKT kinases. A key phenomenon often observed with activation of oncogenes is termed oncogene addiction. This describes a phenomena whereby an activated oncogene drives uncontrolled cellular proliferation, but the cells become “addicted” to the presence of this oncogene, in that survival is dependent on its presence. This can be demonstrated with studies where knockdown of the oncogene, or inhibition of its function via inhibitors, induces cellular death. Oncogene addiction of driver mutations is a key mechanism explored in personalized cancer therapeutics, where the discovery of activated oncogenes in a tumor via sequencing opens up the possibility of targeted therapy for the patient. A third class of mutations key to cancer treatment are resistance mutations. These mutations typically arise during drug treatment and often interfere with the ability for a drug to bind effectively to its target. Drug resistance may also be induced by mutations or alterations to cancer cells that activate other pathways, circumventing the drug’s effectiveness. The final major class of variants found in somatic tumors are passenger, or benign mutations. These variants may induce no important change to cellular behavior, or induce changes to the cell that play no role in promoting tumor formation, growth, survival, or any other hallmarks associated to cancer.</p>
<p>Tumor variants may be hard to detect due to having low allelic fractions and tumor heterogeneity, making variants difficult to distinguish from noise in the assays. Contamination from normal tissue also contributes to this effect. Further difficulties in detection arise from degradation associated to sample storage methods such as when working with formalin-fixed and paraffin-embedded (FFPE) samples.</p>
</div>
<div id="sequencing-background" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Sequencing Background</h2>
<p>Genome sequencing work performed over multiple decades allowed for the first iteration of the human genome reference in the early 2000s<span class="citation">(<a href="#ref-HumanGenome2001" role="doc-biblioref">Eric S. Lander et al. 2001</a>)</span>, followed by sequencing of the first tumor genome<span class="citation">(<a href="#ref-Ley2008" role="doc-biblioref">Ley et al. 2008</a>)</span>.</p>
<p>This work has been greatly aided by the introduction of next generation sequencing (NGS). NGS techniques scale up sequencing to massively parallel processes, numbering in the millions. NGS has largely been performed using techniques based on short-read generation (e.g., 150 bp). These short reads are aligned to a reference genome for analysis and discovery of variants in the sample. New technologies now utilize long multi-kilobase reads, and these platforms can sometimes be used for de novo assembly approaches, without alignment to a reference. These long read techniques allow for long-range haplotype analysis.</p>
<p>Short read technologies typically utilize a sequencing by synthesis (SBS) approach, which involves reading millions of parallel sequencing reactions utilizing fluorescent-labeled nucleotides during the synthesis of a complementary DNA strand to a short fragment. Due to advantages in cost and throughput, short read sequencing is currently predominant in clinical applications. Therefore, clinical workflows to analyze NGS tumor data involve alignment to a reference genome followed by application of variant calling algorithms. Alignment data can be viewed using applications such as IGV (see Figure). Alignment algorithms allow identification of a range of different classes of variants.</p>
<div id="human-genome-reference" class="section level3" number="2.3.1">
<h3><span class="header-section-number">2.3.1</span> Human Genome Reference</h3>
<p>The human genome reference most commonly used, at present, is provided by the Genome Reference Consortium (GRC). The official name for the current GRC human reference genome is Genome Reference Consortium Human Build 38. It is unofficially called hg38 in some resources.</p>
<p>This reference has high base-pair accuracy and good representation of repetitive and segmentally duplicated genomic regions. The assembly was generated with Sanger sequencing methods and large insert BAC clones (&gt;150 kb inserts). DNA from multiple donors was used. The haplotypes from the different donors create different regions whose borders coincide with the underlying clone boundaries.</p>
</div>
</div>
<div id="common-variant-types" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> Common Variant Types</h2>
<p>The variant types usually encountered when sequencing tumors are listed below.</p>
<p>Single nucleotide variants (SNVs) are a change of a single nucleotide in DNA, and are the most common variant found. In cancer, tumor (somatic) SNVs are best detected in comparison to a sample of germline DNA obtained using normal tissue from the same patient. SNVs are also present as natural variation in the germline among individuals and different human populations. Common germline variation is considered to be at a rate of approximately 1% or higher. In protein coding regions, SNVs can result in silent, missense, or nonsense mutations. Silent mutations alter the codon in a way that results in the same amino acid being produced (e.g. CCA and CCG both result in Proline). These mutations usually do not effect cell behavior unless the altered region has a secondary regulatory function. Missense mutations are changes in protein structure that replace one amino acid with another. These mutations are usually denoted with the notation A123B, where A is the original amino acid, 123 is the position of the amino acid in the protein, and B is the new amino acid change. The consequences of missense mutations range from minimal passenger mutations to oncogenic driver mutations (e.g. EGFR L858R). Missense mutations may also introduce premature stop codons (e.g. TGC -&gt; TGA). This results in a nonsense mutation. The consequences of nonsense mutations are usually a large or complete loss of protein function, as all parts of the protein downstream of the premature stop are no longer translated.</p>
<p>Small insertions and deletions (indels) are the second most common form of mutation found both in tumors, and occurring as germline variation in the human population. Indels are usually considered to be insertions and deletions of 50 base pairs or less. In protein coding regions, indels can generate in-frame changes to the amino acid structure of a protein when the insertion or deletion is a multiple of three base pairs, resulting in the addition or loss of one amino acid per three base pairs but preserving the coding frame of downstream bases. Indels can also result in frameshift mutations when the insertion or deletion in a coding region is not a multiple of three. These mutations alter the reading frame, resulting in a radical change to the amino acid sequence downstream of the mutation, usually resulting in protein termination and a non-functional protein product.</p>
<p>Translocations and fusions are a relatively common variant type found in cancer. Translocations are exchanges of large regions of DNA between different chromosomes, or between different regions of the same chromosome. Translocations may have breakpoints that occur in the middle of two different genes, resulting in fusions that have exons from both genes. A well known fusion results from the Philadelphia Chromosome, first identified in 1973 <span class="citation">(<a href="#ref-Rowley1973" role="doc-biblioref">ROWLEY 1973</a>)</span>, which forms when chromosome 9 and chromosome 22 break and exchange large regions. This results in a fusion protein consisting of parts of the BCR gene and the ABL1 kinase gene, including the active kinase domain. The presence of BCR causes tetramer formation, which in turn activates the ABL1 kinase in a disregulated manner, driving oncogenic tyrosine kinase signaling. This fusion drives Chronic myeloid leukemia (CML) and is a driver that occurs in other cancer types as well. Targeted therapies against BCR::ABL1, such as imatinib, have been developed as a result of the discovery of this fusion and its driver properties <span class="citation">(<a href="#ref-Mughal2016" role="doc-biblioref">Mughal et al. 2016</a>)</span>.</p>
<p>A different type of genomic variant consists of copy number variants (CNVs) or copy neutral loss of heterogeneity (CNLOH). Copy number variants consist of losses or gains of large regions of chromosomes. Losses are usually deletions of genomic regions, and gains are usually two or larger fold duplications of genomic regions. These sorts of variants effect large numbers of genes and play a role in cancer. CNLOH usually results from the replacement of a large stretch of DNA on a chromosome with the same stretch of DNA derived from the homologous chromosome in the same cell. This causes a loss of heterozygosity as all DNA in this region is now identical between both homologous chromosomes, but there is no change in copy number of the genes in this region since there were no losses or gains in the DNA of this region.</p>
</div>
<div id="variant-representation-and-classification" class="section level2" number="2.5">
<h2><span class="header-section-number">2.5</span> Variant Representation and Classification</h2>
<p>There are multiple ways to represent, describe, name, and disambiguate variants. Resources like the ClinGen Allele Registry attempt to provide an unambiguous central resource defining particular variants, both at the level of exact DNA changes referencing specific genome build, and at more general levels such as protein changes (e.g. EGFR L858R). The Human Genome Variation Society (HGVS) provides another widely adopted solution, which is a nomenclature for the representation of variants which is both human readable and also offers exact information on the variant.</p>
<p>For germline variant classification, a set of robust processes, data models, and guidelines have been developed. However, with the increased adoption of tumor sequencing in clinical practice, a parallel set of resources for interpretation of somatic variants are equally important.</p>
<p>Systems for classifying germline variant pathogenicity, such as those developed by ACMG and AMP, have been adopted and have influenced subsequent developments for somatic variants. Key differences exist between these two areas of variant interpretation and classification, both in biology and clinical actionability.</p>
<p>In germline variant interpretation, emphasis is placed on the causal relation between the gene’s variants and the associated disease. When these relations are not well defined, clinical action related to the variant is not escalated. In contrast, in somatic variants, while the causal underlying cancer biology of a variant is of clear interest, in clinical application the observation of variant actionability is prioritized above clear knowledge of disease mechanisms. If a variant is shown to induce tumor sensitivity to a certain drug, this will escalate and expand testing of the drug against the variant, and increase clinical adoption of the drug. Studies of the underlying biology will follow and deepen mechanistic understanding and guide the development of further drugs and potential variants, but may not directly influence clinical decisions. Another key difference occurs in the biology of germline and somatic tumor variants. Many somatic variants found in tumors would be embryonic lethal if they occurred in the germline.</p>
<p>In bioinformatics a common file type used for variant representation is the Variant Call Format (VCF) which is a specific file format for storing gene sequence variations. This format developed during large scale sequencing and genotyping projects such as the 1000 Genomes Project or the newer gnomAD database of variant frequencies in the human population <span class="citation">(<a href="#ref-Chen2022" role="doc-biblioref">Chen et al. 2022</a>)</span>.</p>
<p>The VCF format is flexible and allows for representation of many different variant types including those described above. A problem in variant representation in VCF files and other areas of bioinformatics is the non unique way variants can be described. The same variant can be described using reference and alternate alleles of different length and position relative to the reference genome. Different methods attempt to define rules for consistent alignment and parsimony of the variant representation <span class="citation">(<a href="#ref-Tan2015" role="doc-biblioref">Tan, Abecasis, and Kang 2015</a>)</span>. The particular rules for a parsimonious and consistently aligned variant representation are described in various online resources such as this wiki page: <a href="https://genome.sph.umich.edu/wiki/Variant_Normalization" class="uri">https://genome.sph.umich.edu/wiki/Variant_Normalization</a>.</p>

</div>
</div>
<h3> References</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-Chen2022" class="csl-entry">
Chen, Siwei, Laurent C. Francioli, Julia K. Goodrich, Ryan L. Collins, Masahiro Kanai, Qingbo Wang, Jessica Alföldi, et al. 2022. <span>“A Genome-Wide Mutational Constraint Map Quantified from Variation in 76, 156 Human Genomes,”</span> March. <a href="https://doi.org/10.1101/2022.03.20.485034">https://doi.org/10.1101/2022.03.20.485034</a>.
</div>
<div id="ref-HumanGenome2001" class="csl-entry">
Eric S. Lander, and, Lauren M. Linton, Bruce Birren, Chad Nusbaum, Michael C. Zody, Jennifer Baldwin, Keri Devon, et al. 2001. <span>“Initial Sequencing and Analysis of the Human Genome.”</span> <em>Nature</em> 409 (6822): 860–921. <a href="https://doi.org/10.1038/35057062">https://doi.org/10.1038/35057062</a>.
</div>
<div id="ref-Ley2008" class="csl-entry">
Ley, Timothy J., Elaine R. Mardis, Li Ding, Bob Fulton, Michael D. McLellan, Ken Chen, David Dooling, et al. 2008. <span>“<span>DNA</span> Sequencing of a Cytogenetically Normal Acute Myeloid Leukaemia Genome.”</span> <em>Nature</em> 456 (7218): 66–72. <a href="https://doi.org/10.1038/nature07485">https://doi.org/10.1038/nature07485</a>.
</div>
<div id="ref-Mughal2016" class="csl-entry">
Mughal, T. I., J. P. Radich, M. W. Deininger, J. F. Apperley, T. P. Hughes, C. J. Harrison, C. Gambacorti-Passerini, G. Saglio, J. Cortes, and G. Q. Daley. 2016. <span>“Chronic Myeloid Leukemia: Reminiscences and Dreams.”</span> <em>Haematologica</em> 101 (5): 541–58. <a href="https://doi.org/10.3324/haematol.2015.139337">https://doi.org/10.3324/haematol.2015.139337</a>.
</div>
<div id="ref-Rowley1973" class="csl-entry">
ROWLEY, JANET D. 1973. <span>“A New Consistent Chromosomal Abnormality in Chronic Myelogenous Leukaemia Identified by Quinacrine Fluorescence and Giemsa Staining.”</span> <em>Nature</em> 243 (5405): 290–93. <a href="https://doi.org/10.1038/243290a0">https://doi.org/10.1038/243290a0</a>.
</div>
<div id="ref-Tan2015" class="csl-entry">
Tan, Adrian, Gonçalo R. Abecasis, and Hyun Min Kang. 2015. <span>“Unified Representation of Genetic Variants.”</span> <em>Bioinformatics</em> 31 (13): 2202–4. <a href="https://doi.org/10.1093/bioinformatics/btv112">https://doi.org/10.1093/bioinformatics/btv112</a>.
</div>
</div>
<hr>
<center> 
  <div class="footer">
      All illustrations <a href="https://creativecommons.org/licenses/by/4.0/">CC-BY. </a>
      <br>
      All other materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="introduction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="introduction-to-somatic-evidence-types-in-cancer.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
